edgeworkers/Acrobat_DC_web_prod/utils/csp/index.js

```diff
@@ -0,0 +1,18 @@
+import stage from './stage.js';
```

Comment:
is there a way we can link everything in the csp/ folder to https://github.com/adobecom/dc/tree/stage/acrobat/scripts/contentSecurityPolicy